docs/reference/troubleshooting/common-issues/task-queue-backlog.asciidoc

```diff
@@ -1,103 +1,117 @@
 [[task-queue-backlog]]
 === Task queue backlog
 
-A backlogged task queue can prevent tasks from completing and put the cluster
-into an unhealthy state. Resource constraints, a large number of tasks being
-triggered at once, and long running tasks can all contribute to a backlogged
-task queue.
+*******************************
+*Product:* Elasticsearch +
+*Deployment type:* Elastic Cloud (hosted or self-managed), self-managed +
```

Comment:
given that this tutorial uses cat thread pool apis it's def not for serverless ... but probably for everything else

Reply:
I wonder whether there's a case for just saying "All types except serverless" -- seems like that might be the case often enough?

Reply:
:2c: until we clean up our usage of elastic cloud it's better to be specific (to help later contributors)

Reply:
(or do me a favor and add it as a comment)

Reply:
yeah I think we need to see how some conversations around that official messaging goes - but agree we should be able to link to a source of truth. afraid of putting this list together now because it is actively being discussed